.github/workflows/pr.yaml

```
@@ -44,6 +44,22 @@ jobs:
           do
             sleep 1
           done
+      - name: Deploy konk-service
+        timeout-minutes: 5
+        run: |
+          make deploy-konk-service
+      - name: Deploy konk-service-operator
+        timeout-minutes: 5
+        run: |
+          make deploy-konk-service-operator
+          make deploy-example-konk-service
+          until kubectl wait --timeout=3m --for=condition=ready pod -l app.kubernetes.io/component=apiservice,app.kubernetes.io/instance=example-konkservice
+          do
+            sleep 1
+          done
+          POD_NAME=$(kubectl get pod -o name | grep konkservice)
+          kubectl exec -it $POD_NAME -- kubectl get apiservice
+          kubectl exec -it $POD_NAME -- kubectl get crd
```

You're on the right track here, thanks.

This will still pass if the apiservice is not installed correctly, right? How about this:
```suggestion
          kubectl exec -it $POD_NAME -- kubectl get apiservice v1alpha1.test.infoblox.com
          kubectl exec -it $POD_NAME -- kubectl get crd appconfigs.stable.example.com
```

---

Done, thanks!